components/compliance-service/ingest/server/compliance.go

```diff
@@ -197,14 +203,39 @@ func (s *ComplianceIngestServer) handleNotifications(ctx context.Context, report
 	return nil
 }
 
-func (s *ComplianceIngestServer) sendNodeInfoToManager(ctx context.Context, node manager.NodeMetadata) error {
+func (s *ComplianceIngestServer) sendNodeInfoToManager(ctx context.Context, node manager.NodeMetadata, profiles []*ingest_inspec.Profile) error {
 	if s.mgrClient == nil {
 		return fmt.Errorf("no manager client found")
 	}
+	node.ScanData.Status = getReportStatus(profiles)
 	logrus.Debugf("sendNodeInfoToManager handing-over node to manager %+v", s)
 	_, err := s.mgrClient.ProcessNode(ctx, &node)
 	if err != nil {
 		return errors.Wrap(err, "sendNodeInfoToManager error calling ProcessNode")
 	}
 	return nil
 }
+
+func getReportStatus(profiles []*ingest_inspec.Profile) (status string) {
+	// start with a status of passed as the default status
+	status = "passed"
+	skippedCounter := 0
+	for _, profile := range profiles {
+		// if any profile is failed, report is failed
+		if profile.Status == "failed" {
```

Comment:
i just realizeed this logic if faulty. profiles will have a status of skipped if skipped for a platform exception, but will not otherwise have the status. need to look at the controls to get that

Reply:
this logic is fixed now